### DISCUSSION TITLE
Contact

### DISCUSSION THREAD

**Discussion by Jvr2022** [xanonDev/pinkcord#49] [General]
Hi @xanonDev my discord got banned and i have a new account. Please contact me with speedygamerx_alt

**Comment by ShamesFul** [comment]
Hey Xanon could I get a discord name or something to contact on? Or just chuck it some where in the Github.  Got some issues that I could DM you about :)

**Reply by Jvr2022** [reply]
Xanon left discord

**Reply by ShamesFul** [reply]
Is there a way to contact?

**Reply by Jvr2022** [reply]
Yes, you can contact me on discord because I am the only one that work on the tool now @ShamesFul user: jvr2022_github
